src/PantheonTerminalWindow.vala

```diff
@@ -325,6 +325,14 @@ namespace PantheonTerminal {
             default_width = PantheonTerminal.saved_state.window_width;
             default_height = PantheonTerminal.saved_state.window_height;
 
+            if (default_width == -1) {
+                default_width = Gdk.Screen.width () * 2 / 3;
```

Comment:
Works on my hidpi display. For multiple screen maybe it needs work but existing move x y code might already have this issue anyway.